Dockerfile

```diff
@@ -20,4 +20,4 @@ COPY . .
 RUN yarn build
 
 EXPOSE 3000
-CMD [ "./node_modules/.bin/forever", "lib/index.js" ]
```

Comment:
what does `forever` actually give us now that we'll be losing?

Reply:
Forever just makes sure to restart the the process if it exits. However, kubernetes does this at the container level and is the more standard way of doing it.